tasks/config-gitlab-runner.yml

```diff
@@ -8,10 +8,35 @@
     backrefs: True
   notify: restart gitlab-runner
   loop:
-    - {param: "concurrent", value: "{{ global_section.concurrent }}"}
+    - {param: 'concurrent', value: '{{ global_section.concurrent }}'}
```

Comment:
Why not map syntax?
```yaml
    - param: concurrent
      value: '{{ global_section.concurrent }}'
```

Reply:
Dictionaries and lists can also be represented in an abbreviated form,
These are called “Flow collections”.
https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html

Reply:
Simple key-value syntax is better readable here, isn't it?